### DISCUSSION TITLE
Modify inbound email HTML to include SL handling info at the top (or bottom) of the <BODY>

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#599] [Feature Requests]
The nature of the SimpleLogin service means that all inbound messages destined for a user pass through an SL server first. As such SL has the ability to modify the HTML (an awesome responsibility!) This feature request is to **optionally** add a set of standardized information, controllable through SL Settings, at the top (or bottom) of each message's HTML <BODY> section that shows:

1. The From email address (both friendly name and email address),
2. The SL Alias the message was directed to,
2. The SL Mailbox(es) you forwarded the message to, and
3. A link (or button) to disable the SL Alias, if desired.

The goal of this feature is twofold. First, it allows the routing of the message to be more easily understood by the recipient (e.g. "Where did this thing come from?" and "Did it get forwarded to my wife/husband/dog _too?_") And second, this provides the equivalent of a one-click unsubscribe capability within an inbound email that does not necessitate going to the SL website, searching for the Alias, etc.

I strongly recommend that this inclusion of this capability be user selectable as an option in the system settings. I believe the default condition should be turned off (though I, for one, will really like having the capability and would definitely turn it on)!

**Comment by WhamboMPS** [comment]
@nguyenkims, I just received a messaged that had the anti-phishing warning in it. That feature basically uses the same HTML-editing feature that I requested in this post. Any chance this request will actually be implemented?

**Reply by nguyenkims** [reply]
@WhamboMPS now that we found a way to modify email body without affecting its formatting, we can indeed implement this feature. Currently our roadmap is quite packed and I don't know when we'll get to this feature though, hopefully soon!
